src/jit/lower.cpp

```diff
@@ -5819,6 +5819,12 @@ void Lowering::ContainCheckRet(GenTreeOp* ret)
             {
                 MakeSrcContained(ret, op1);
             }
+#if defined(_TARGET_ARM64_) && defined(FEATURE_SIMD)
+            if (varTypeIsSIMD(op1))
+            {
+                MakeSrcContained(ret, op1);
+            }
+#endif // defined(_TARGET_ARM64_) && defined(FEATURE_SIMD)
```

Comment:
It won't work to make a `TYP_SIMD` lclVar contained under a return if it is enregistered. A contained reference won't be observed by the register allocator, and therefore the register allocator will not ensure that it is in a register at this point. And, if it is a register candidate, it may not be live on the stack either. I don't know if we have such an assert, but there should probably be an assert somewhere that a register-candidate lclVar is never contained.

Reply:
Sounds like this patch needs to be revised/abandoned....

I am still debugging various asserts, so this could be the root cause of one or more....